kivymd/uix/button/button.py

```diff
@@ -514,6 +514,16 @@ def build(self):
     Builder.load_string(kv_file.read())
 
 
+THEME_TEXT_COLOR_OPTIONS = [
```

Comment:
You don't need to use capital letters.

Reply:
I was following the PEP8 guidelines on module-level constants (all caps), although you can make the argument a list isn't a 'constant' in the usual sense. In any event, will change to lowercase (and should really be a tuple rather than a list).